scripts/deploy-to-gcs.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+#set -x
+
+set -eu
+
+# Publish the helm charts to GCS.  This is primarily intended to be run by Travis but is
+# usable standalone (assuming you have docker installed and are running from the helm-charts directory).
+
+# Use caution with the -d flag.  If you happen to run this against the production repo but from a local
+# helm-charts dir with several tarballs missing for some reason, those tarballs will be removed from the
+# production repo.  Deletion is disabled by default.
+
+# This implementation inspired by https://medium.com/google-cloud/google-cloud-sdk-dockerfile-861a0399bbbb
+
+usage () {
+    echo "${0##*/} [-d] [-n] | [-h]"
+    echo "  -d   Delete objects in repo that are not in docs dir.  Default is false."
+    echo "  -n   Dry run.  Show what would be published.  Default is false"
```

Comment:
I would recommend confirmation by default and a force flag [-y] to override

Reply:
I'd rather not, since this script could be called in a non-interactive fashion from Travis if we choose to go that route.  I guess I could check if the shell is interactive (or for a Travis env var).  Will consider...

Reply:
what about executing -n dry run always so that we can later look at logs to see what got synced

Reply:
Not sure I follow.  If you don't specify `-n`, the logs will still contain a list of what got synced.  See for e.g., https://github.com/lightbend/helm-charts/pull/96, where I artificially tweaked the `index.yaml` file to see if (only) it got picked up correctly.

Or did you mean something else?

Reply:
If the logs contain a list of files that got synced.. thats all I need thanks.